modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {SMProductName} and Kiali
+
+//Kiali attributes are in the works, and possible {KialiProduct} will refer to Kiali provided by Red Hat, and not Kiali Operator provided by Red Hat.
+//Kiali attributes are outside the scope of this PR.
+//TP1 content. IA influx, likely everything will change for GA.
+
+Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali provided by Red Hat is composed of three parts:
```

Comment:
Per [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute): When pointing to a link that exists external to Red Hat’s customer portal documentation, add the link attribute, and use the <document_heading_name> (<document_source>) format.